### ISSUE TITLE
Make Unit a sub-element of Property

### ISSUE THREAD

**Issue by NightDog** [microrealestate/microrealestate#10]
When managing apartments or houses a Property might have multiple Units that can be rented out.

Some examples:

- Apartment building with multiple units.
- House that is split, and rented out as multiple units.
- Condo that has a garage spot attached, and might be rented out to a different person than the tenants.

These Units would be of different types. They can be Condominium, Room, Parking Spot, or other types.

What they all have in common is that they are attached to a Lease, Tenant, and they are all part of a parent Property.

**Comment by camelaissani** [comment]
Good point! The concept of parent Property is missing and might be helpful to organize the different units.

BTW, I have started to rewrite the UI with React and Nextjs [here](https://github.com/microrealestate/frontend) 
Currently, the [Property page](https://github.com/microrealestate/frontend/blob/master/src/pages/%5Borganization%5D/estates.js) is still blank. 
Do you have any suggestions on how we could show the properties and which data could be displayed there?

**Comment by NightDog** [comment]
Did a little mock-up of what a page for "Property" might look like with some comments:
<img width="1409" alt="Skjermbilde 2021-02-28 kl  14 00 06" src="https://user-images.githubusercontent.com/3144364/109419352-50747c80-79cd-11eb-87e7-8d2cdbec9000.png">

A little bit unsure about how to display this in a way that makes it accessable on mobile devices as well. Giving enough information, but still not over-doing it.
